packages/node-cli/src/commands/ManageAccountCommandGenerator.js

```diff
@@ -0,0 +1,283 @@
+			};
+		} else if (answers[ANSWERS_NAMES.ACTION] == MANAGE_ACTION.REMOVE) {
+			return {
+				[COMMAND.OPTIONS.REMOVE]: answers[ANSWERS_NAMES.SELECTED_AUTH_ID].authId,
+			};
+		}
+	}
+
+	async _executeAction(answers) {
+		const sdkParams = CommandUtils.extractCommandOptions(answers, this._commandMetadata);
+		const flags = [];
+		if (answers[COMMAND.OPTIONS.LIST]) {
+			flags.push(COMMAND.OPTIONS.LIST);
+			delete sdkParams[COMMAND.OPTIONS.LIST];
+		}
+		const executionContext = new SdkExecutionContext({
+			command: this._commandMetadata.sdkCommand,
+			params: sdkParams,
+			flags,
+		});
+
+		const selectedAction = this._extractExecutedAction(answers);
+		const authId = this._extractAuthId(answers);
+		const message = this._getSpinnerMessage(selectedAction, authId);
+
+		const operationResult = await executeWithSpinner({
+			action: this._sdkExecutor.execute(executionContext),
+			message: message,
+		});
+
+		return operationResult.status === SdkOperationResultUtils.STATUS.SUCCESS
+			? ManageAccountActionResult.Builder.withData(this._prepareData(selectedAction, operationResult.data))
+					.withResultMessage(operationResult.resultMessage)
+					.withExecutedAction(selectedAction)
+					.build()
+			: ManageAccountActionResult.Builder.withErrors(SdkOperationResultUtils.collectErrorMessages(operationResult)).build();
+	}
+
+	_getSpinnerMessage(action, authId) {
+		switch (action) {
+			case MANAGE_ACTION.REMOVE:
+				return NodeTranslationService.getMessage(MESSAGES.REMOVING);
+			case MANAGE_ACTION.RENAME:
+				return NodeTranslationService.getMessage(MESSAGES.RENAMING);
+			case MANAGE_ACTION.LIST:
+				return NodeTranslationService.getMessage(MESSAGES.LISTING);
+			case MANAGE_ACTION.REVOKE:
+				return NodeTranslationService.getMessage(MESSAGES.REVOKING);
+			case MANAGE_ACTION.INFO:
+				return NodeTranslationService.getMessage(MESSAGES.INFO, authId);
+		}
+		assert.fail(NodeTranslationService.getMessage(ERRORS.UNKNOWN_ACTION));
+	}
+
+	_prepareData(action, data) {
+		if (action != MANAGE_ACTION.INFO) {
+			return data;
+		}
+		assert(this._authId);
+		assert(data.hasOwnProperty(DATA_PROPERTIES.ACCOUNT_INFO));
+		let actionResultData = { authId: this._authId, accountInfo: data.accountInfo };
+		if (data.hasOwnProperty(DATA_PROPERTIES.URLS)) {
+			actionResultData[DOMAIN] = data.urls.app;
+		}
+		return actionResultData;
+	}
+
+	_extractExecutedAction(answers) {
+		if (answers.hasOwnProperty(COMMAND.OPTIONS.INFO)) {
+			this._authId = answers[COMMAND.OPTIONS.INFO];
+			return MANAGE_ACTION.INFO;
+		}
+		if (answers.hasOwnProperty(COMMAND.OPTIONS.LIST)) {
+			return MANAGE_ACTION.LIST;
+		}
+		if (answers.hasOwnProperty(COMMAND.OPTIONS.REMOVE)) {
+			return MANAGE_ACTION.REMOVE;
+		}
+		if (answers.hasOwnProperty(COMMAND.OPTIONS.RENAME)) {
+			return MANAGE_ACTION.RENAME;
+		}
+		assert.fail(NodeTranslationService.getMessage(ERRORS.UNKNOWN_ACTION));
+	}
+
+	_extractAuthId(answers) {
```

Comment:
There's code duplication in _extractSelectedActipon and _extractAuthId. I'd suggest to combine them and retun one object that can be later used, e.g. { "selectedAction" : MANAGE_ACTION.INFO,  "authId" : "myid" }